.github/workflows/test.yml

```diff
@@ -9,6 +9,17 @@ on:
       - '.gitignore'
 
 jobs:
+  bash-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run bash tests
+        shell: bash
+        run: |
+          echo "Running bash tests..."
```

Comment:
🔍 Ensure that 'tests/test_action_checks.sh' exists and is executable to avoid runtime errors.